example/lottieview.h

```diff
@@ -66,8 +66,8 @@ class RenderStrategy {
     }
     virtual void resize(int width, int height) = 0;
     virtual void setPos(int x, int y) {evas_object_move(renderObject(), x, y);}
-    virtual size_t findFrameAtMarker(const std::string &markerName) = 0;
-    virtual size_t findDurationFrameAtMarker(const std::string &markerName) = 0;
+    virtual size_t findStartFrameAtMarker(const std::string &markerName) = 0;
```

Comment:
Can you create a structure which will return the duration and start frame so that the code will be in one function

Reply:
I updated patch. 
I change return type size_t to tuple<size_t, size_t>